lib/system/atomics.nim

```diff
@@ -211,26 +211,38 @@ elif someVcc and hasThreadSupport:
         importc: "_InterlockedExchangeAdd", header: "<intrin.h>".}
 
 else:
+  type AtomMemModel* = distinct cint
+
+  const
+    ATOMIC_RELAXED = 0.AtomMemModel
+    ATOMIC_CONSUME = 1.AtomMemModel
+    ATOMIC_ACQUIRE = 2.AtomMemModel
+    ATOMIC_RELEASE = 3.AtomMemModel
+    ATOMIC_ACQ_REL = 4.AtomMemModel
+    ATOMIC_SEQ_CST = 5.AtomMemModel
```

Comment:
These should be exported, I think? The `ATOMIC_` constants in the `vcc` branch should probably be too.

Reply:
Yeah.  I asked in IRC but I figured that this would be the simplest and least impactful change.  But I cannot think of a reason not to expose the type fully; especially since we do allow invocation of the inc/dec without regard to threading...

Reply:
this can be covered later, I say merge and revisit if/when needed.